src/main.rs

```diff
@@ -267,7 +267,11 @@ impl PluginifyCommand {
                 let mut tar_builder = tar::Builder::new(&mut enc);
                 tar_builder.append_path_with_name(&package, &name)?;
                 for asset in ps.assets() {
-                    tar_builder.append_path(asset).with_context(|| format!("Adding {}", asset.display()))?;
+                    if asset.is_dir() {
+                        tar_builder.append_dir_all(asset, asset)?;
```

Comment:
needs moar `with_context` as per the file case

Reply:
updated :)